example/linalg/example_eig.f90

```diff
@@ -0,0 +1,27 @@
+! Eigendecomposition of a real square matrix 
+program example_eig
+  use stdlib_linalg_constants, only: sp
```

Comment:
I suggest to not mention `sp`, as it is the default (except if some compiler options are turned on).
```suggestion
```

Reply:
Fixed at [cdef45f](https://github.com/fortran-lang/stdlib/pull/816/commits/cdef45f142bc40da7599574c87fe554088298901)